app/models/index-card.ts

```diff
@@ -90,15 +90,23 @@ export default class IndexCardModel extends Model {
     async getOsfModel(options?: object) {
         const identifier = this.resourceIdentifier;
         if (identifier && this.osfModelType) {
-            const guid = this.guidFromIdentifierList();
+            const guid = this.guidFromIdentifierList;
             if (guid) {
                 const osfModel = await this.store.findRecord(this.osfModelType, guid, options);
                 this.osfModel = osfModel;
             }
         }
     }
 
-    guidFromIdentifierList() {
+    get osfUrl() {
+        const guid = this.guidFromIdentifierList;
```

Comment:
nit: these two `osfUrl` and `guidFromIdentifierList` could both be simplified if reversed -- one getter to go thru the identifier list and return the one that starts with the expected prefix, then another getter to strip that expected prefix